src/fragment-components/a-inline-loading.tsx

```diff
@@ -0,0 +1,238 @@
+	...rest
+}: any) => {
+	const status: any = {
+		active: {
+			iconDescription: componentObj.activeIconDescription,
+			description: componentObj.activeText
+		},
+		error: {
+			iconDescription: componentObj.errorIconDescription,
+			description: componentObj.errorText
+		},
+		inactive: {
+			iconDescription: componentObj.inactiveIconDescription,
+			description: componentObj.inactiveText
+		},
+		finished: {
+			iconDescription: componentObj.finishedIconDescription,
+			description: componentObj.successText
+		}
+	};
+	return (
+		<AComponent
+		componentObj={componentObj}
+		headingCss={css`display: block;`}
+		rejectDrop={true}
+		{...rest}>
+			<InlineLoading
+			description={status[componentObj.status].description}
+			iconDescription={status[componentObj.status].iconDescription}
+			status={componentObj.status}
+			className={componentObj.cssClasses?.map((cc: any) => cc.id).join(' ')} />
+		</AComponent>
+	);
+};
+
+export const componentInfo: ComponentInfo = {
+	component: AInlineLoading,
+	codeUI: AInlineLoadingCodeUI,
+	settingsUI: AInlineLoadingSettingsUI,
+	keywords: ['inline', 'loading'],
+	name: 'Inline Loading',
+	type: 'inline-loading',
+	defaultComponentObj: {
+		type: 'inline-loading',
+		status: 'active',
+		activeText: '',
+		inactiveText: '',
+		successText: '',
+		errorText: '',
+		activeIconDescription: '',
+		errorIconDescription: '',
+		finishedIconDescription: '',
+		inactiveIconDescription: ''
+	},
+	image,
+	codeExport: {
+		angular: {
+			inputs: ({ json }) => `@Input() ${nameStringToVariableString(json.codeContext?.name)}Status = "${json.status}";
+				@Input() ${nameStringToVariableString(json.codeContext?.name)}LoadingText = "${json.activeText}";
+				@Input() ${nameStringToVariableString(json.codeContext?.name)}SuccessText = "${json.successText}";
+				@Input() ${nameStringToVariableString(json.codeContext?.name)}ErrorText = "${json.errorText}";`,
+			outputs: ({ json }) => `@Output() ${nameStringToVariableString(json.codeContext?.name)}OnSuccess = new EventEmitter();`,
+			imports: ['InlineLoadingModule'],
+			code: ({ json }) => {
+				return `<ibm-inline-loading
+					(onSuccess)="${nameStringToVariableString(json.codeContext?.name)}OnSuccess.emit($event)"
+					[state]="${nameStringToVariableString(json.codeContext?.name)}Status"
+					[loadingText]="${nameStringToVariableString(json.codeContext?.name)}LoadingText"
+					[successText]="${nameStringToVariableString(json.codeContext?.name)}SuccessText"
+					[errorText]="${nameStringToVariableString(json.codeContext?.name)}ErrorText"
+					${angularClassNamesFromComponentObj(json)}>
+				</ibm-inline-loading>`;
+			}
+		},
+		react: {
+			imports: ['InlineLoading'],
+			code: ({ json }) => {
+				const status = `state["${nameStringToVariableString(json.codeContext?.name)}"] || "${json.status}"`;
+				return `<InlineLoading
+					onSuccess={(success) => handleInputChange({
+						target: {
+							name: "${json.codeContext?.name}",
+						}
+					})}
```

Comment:
@zvonimirfras 

I know for a fact this isn't correct, the idea behind on success is to call a function when status is `finished`. But the users are controlling the status, so I don't see a reason in having this onSuccess function at all, should we just remove it or do a check where we expect users will be passing in a function to execute on `finished`?

Other then this one change, this PR looks good to me, let me know what you think we should do here.